src/js/clickable-component.js

```diff
@@ -67,9 +82,11 @@ class ClickableComponent extends Component {
   /**
    * create control text
    *
-   * @param {Element} el Parent element for the control text
+   * @param {Element} [el]
+   *        Parent element for the control text
+   *
    * @return {Element}
-   * @method controlText
+   *         the control text element
    */
```

Comment:
This could probably stand to have some capitalization added to both the `@return` and the description above.